packages/xo-web/src/xo-app/backup-ng/index.js

```diff
@@ -430,8 +430,10 @@ export default routes('overview', {
   'restore/metadata': RestoreMetadata,
   'file-restore': FileRestore,
   health: Health,
-})(({ children }) => (
-  <Page header={HEADER} title='backupPage' formatTitle>
-    {children}
-  </Page>
-))
+})(
+  adminOnly(({ children }) => (
```

Comment:
I think, this should be the topmost decorator (for performance and security concerns).

Reply:
Why is it a security issue?

Reply:
It just ensure that nothing (no other decorators) is processed before the permission is checked.